docs/README.md

```diff
@@ -46,12 +47,12 @@ This example expects an NVIDIA GPU by default to simulate with k-Wave.
 
 To test the reconstruction on a machine with a GPU,
 set `RUN_SIMULATION` [on line 14 of `bmode_reconstruction_example.py`](https://github.com/waltsims/k-wave-python/blob/master/examples/bmode_reconstruction_example.py#L18)
-to `True` and the example will run without the pre-computed data.
+to `True`, and the example will run without the pre-computed data.
 
 ## Development
 
 If you're enjoying k-Wave-python and want to contribute, development instructions can be
-found [here](https://waltersimson.com/k-wave-python/development/development_environment.html).
+found [here](https://k-wave-python.readthedocs.io/en/latest/development/development_environment.html).
```

Comment:
If I remember correctly, you wanted to use your domain for redirecting to the docs page?